energyplus_regressions/tests/resources/eplusout_perflog_base.csv

```diff
@@ -1,2 +1,2 @@
-Program, Version, TimeStamp,Use Coil Direct Solution,Zone Radiant Exchange Algorithm,Override Mode,Number of Timesteps per Hour,Minimum Number of Warmup Days,SuppressAllBeginEnvironmentResets,Minimum System Timestep,MaxZoneTempDiff,MaxAllowedDelTemp,Electricity ABUPS Total [J],Natural Gas ABUPS Total [J],Gasoline ABUPS Total [J],Diesel ABUPS Total [J],Coal ABUPS Total [J],Fuel Oil No 1 ABUPS Total [J],Fuel Oil No 2 ABUPS Total [J],Propane ABUPS Total [J],Other Fuel 1 ABUPS Total [J],Other Fuel 2 ABUPS Total [J],District Cooling ABUPS Total [J],District Heating ABUPS Total [J],Water ABUPS Total [m3],Values Gathered Over [hours],Facility Any Zone Oscillating Temperatures Time [hours],Facility Any Zone Oscillating Temperatures During Occupancy Time [hours],Facility Any Zone Oscillating Temperatures in Deadband Time [hours],Run Time [seconds],Run Time [string],Number of Warnings,Number of Severe,
-EnergyPlus, Version 9.4.0-734af19f9a, YMD=2020.12.02 13:45,False,CarrollMRT,Normal,6,1,False,2.0,0.30,2.0000E-003,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.000,0.00,16.32,7.58,0.00,17.66,00hr 00min 17.66sec,6,0,
+Program, Version, TimeStamp,Use Coil Direct Solution,Zone Radiant Exchange Algorithm,Override Mode,Number of Timesteps per Hour,Minimum Number of Warmup Days,SuppressAllBeginEnvironmentResets,Minimum System Timestep,MaxZoneTempDiff,MaxAllowedDelTemp,Electricity ABUPS Total [J],Natural Gas ABUPS Total [J],Gasoline ABUPS Total [J],Diesel ABUPS Total [J],Coal ABUPS Total [J],Fuel Oil No 1 ABUPS Total [J],Fuel Oil No 2 ABUPS Total [J],Propane ABUPS Total [J],Other Fuel 1 ABUPS Total [J],Other Fuel 2 ABUPS Total [J],District Cooling ABUPS Total [J],District Heating Water ABUPS Total [J],District Heating Steam ABUPS Total [J],Water ABUPS Total [m3],Values Gathered Over [hours],Facility Any Zone Oscillating Temperatures Time [hours],Facility Any Zone Oscillating Temperatures During Occupancy Time [hours],Facility Any Zone Oscillating Temperatures in Deadband Time [hours],Run Time [seconds],Run Time [string],Number of Warnings,Number of Severe,
```

Comment:
Updated all 3 fake perflog test files to account for the new token.  Tests pass now, confirming we are ignoring the correct timestamp tokens.